multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -157,6 +200,13 @@ write_files:
           "http://127.0.0.1:8080/api/v1/namespaces/kube-system/services"
       done
 
+  - path: /etc/rkthack
```

Comment:
nit: `/opt/bin` or `/usr/local/bin`? etc seems like an odd place. Also maybe `rkt-hostns-wrapper` or something like that? rkthack isn't very descriptive -- and it's not a hack -- it's a "root namespace entry solution" or a "system namespace re-entry procedure"

/cc @philips